javascript/builtins/WebAssembly.json

```diff
@@ -0,0 +1,1350 @@
+{
+  "javascript": {
+    "builtins": {
+      "WebAssembly": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly",
+          "support": {
+            "webview_android": {
+              "version_added": "57"
+            },
+            "chrome": {
+              "version_added": "57"
+            },
+            "chrome_android": {
+              "version_added": "57"
+            },
+            "edge": {
+              "version_added": "16"
+            },
+            "edge_mobile": {
+              "version_added": true
```

Comment:
I've got this listed as no on the existing compat tables, but their platform status page has it listed as a preview in Edge mobile (https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webassemblymvp/). Doesn't this mean it's preffed off though?

Reply:
Yeah, good point, I think we should indicate mobile support behind a pref still. Will update.